.github/workflows/release.yaml

```diff
@@ -18,35 +18,105 @@
-        uses: docker/setup-buildx-action@v3
+      - name: Create the PR to bump the version in the main branch (only if we're tagging from main branch)
+        if: ${{ github.event.inputs.branch == 'main' }}
+        run: |
+          git config --local user.name ${{ github.actor }}
+          CURRENT_VERSION=$(echo "${{ steps.TAG_UTIL.outputs.extVersion }}")
+          tmp=${CURRENT_VERSION%.*}
+          minor=${tmp#*.}
+          bumpedVersion=${CURRENT_VERSION%%.*}.$((minor + 1)).0
+          bumpedBranchName="bump-to-${bumpedVersion}"
+          git checkout -b "${bumpedBranchName}"
+          sed -i  "s#version\":\ \"\(.*\)\",#version\":\ \"${bumpedVersion}-next\",#g" package.json
+          git add package.json
+          git commit -s --amend -m "chore: bump version to ${bumpedVersion}"
+          git push origin "${bumpedBranchName}"
+          echo -e "📢 Bump version to ${bumpedVersion}\n\n${{ steps.TAG_UTIL.outputs.extVersion }} has been released.\n\n Time to switch to the new ${bumpedVersion} version 🥳" > /tmp/pr-title
+          pullRequestUrl=$(gh pr create --title "chore: 📢 Bump version to ${bumpedVersion}" --body-file /tmp/pr-title --head "${bumpedBranchName}" --base "main")
+          echo "📢 Pull request created: ${pullRequestUrl}"
+          echo "➡️ Flag the PR as being ready for review"
+          gh pr ready "${pullRequestUrl}"
+          echo "🔅 Mark the PR as being ok to be merged automatically"
+          gh pr merge "${pullRequestUrl}" --auto --rebase
+          git checkout ${{ steps.TAG_UTIL.outputs.githubTag }}
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
 
+  build:
+    needs: [tag]
+    runs-on: ubuntu-latest
+    steps:
       - uses: actions/checkout@v4
+        with:
+          ref: ${{ needs.tag.outputs.githubTag }}
 
       - name: Login to ghcr.io
-        uses: docker/login-action@v3
-        with:
-          registry: ghcr.io
-          username: ${{ github.repository_owner }}
-          password: ${{ secrets.GITHUB_TOKEN }}
+        run: podman login --username ${{ github.repository_owner }} --password ${{ secrets.GITHUB_TOKEN }} ghcr.io
 
       - name: Build Image
         id: build-image
-        uses: docker/build-push-action@v6
-        with:
-          context: .
-          file: oci/Containerfile.multistage
-          platforms: linux/amd64,linux/arm64
-          push: true
-          tags: ghcr.io/crc-org/crc-extension:latest
+        run: |
+          podman build -t ghcr.io/${{ github.repository_owner }}/crc-extension:${{ needs.tag.outputs.extVersion }} . -f ./oci/Containerfile.multistage
+          podman push ghcr.io/${{ github.repository_owner }}/crc-extension:${{ needs.tag.outputs.extVersion }}
+          podman tag ghcr.io/${{ github.repository_owner }}/crc-extension:${{ needs.tag.outputs.extVersion }} ghcr.io/${{ github.repository_owner }}/crc-extension:latest
+          podman push ghcr.io/${{ github.repository_owner }}/crc-extension:latest
```

Comment:
it seems the build for a different platform is gone (I don't know if for example the extension embeds some binaries specific to a platform at build time ?)

Reply:
I removed it because the extension does not have platform specific code